docs/admin/auth/methods.rst

```diff
@@ -108,6 +108,47 @@ applies to the PostgreSQL wire protocol, as there the username isn't optional.
 Please consult the relevant client documentations for instructions on how to
 connect using SSL with client certificate.
 
+.. _auth_jwt:
+
+JWT authentication method
+=========================
+
+This method is useful to obtain temporary access to CrateDB without sharing
+credentials of a specific user.
+One service is an authentication provider, responsible for issuing tokens which
+can be used by other services to access CrateDB. The authentication provider
+signs issued tokens with a private key and exposes public key(s) in a
+`JWK endpoint`_.
+
+When the ``jwt`` authentication method is used, the client has to provide a
+``BASE64_`` encoded `JSON Web Token`_ (JWT),  prefixed with ``Bearer``, in the
+``Authorization`` HTTP header.
```

Comment:
How about something more like this? The first two sentences are kinda hard to understand, especially if you don't already know what it is or how it is used.


```suggestion
JWT authentication allows to delegate part of the authentication process to an
external service.

The external service is responsible for issuing a ``JWT`` access token for the
user. The user then provides this token as part of a ``Authorization`` HTTP
header to CrateDB.
CrateDB will validate the token and match it to a user created with ``CREATE
USER`` with ``JWT`` properties that match those of the provided ``JWT`` token.

```

Reply:
Applied the suggestion but added removed links in https://github.com/crate/crate/pull/15551/commits/6e465707d3086074a1fadb351553fc19b19632e8

I think it's fair to mention "bearer" (same as with do with basic) and add some rfc links to JWT and JWK so that users who setup the endpoint/auth provider themselves  can check the format.